src/SanitizeTimestampCapableTrait.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace RebelCode\Time;
+
+use Dhii\Util\String\StringableInterface;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception as RootException;
+use InvalidArgumentException;
+
+/**
+ * Common functionality for objects that can sanitize timestamps.
+ *
+ * @since [*next-version*]
+ */
+trait SanitizeTimestampCapableTrait
+{
+    /**
+     * Sanitizes a timestamp to an integer number.
+     *
+     * @since [*next-version*]
+     *
+     * @param int|string|StringableInterface|null $timestamp The timestamp.
+     *
+     * @return int The sanitized timestamp.
+     */
+    protected function _sanitizeTimestamp($timestamp)
```

Comment:
Why not call [`_normalizeInt()`] here?

[`_normalizeInt()`]: https://github.com/Dhii/normalization-helper-base/blob/task/initial-classes/src/NormalizeIntCapableTrait.php#L30

Reply:
I don't think `_normalizeInt()` was available at the time of writing this. I will change it.

Reply:
Which sounds better? Having `_sanitizeTimestamp()` call an abstract `_normalizeInt()`? Or replacing all uses of `_sanitizeTimestamp()` with `_normalizeInt()`?

Reply:
I'm tending towards replacing all uses with `_normalizeInt()`. While there _may_ be a benefit to having an extra layer of abstraction, these properties (duration, timestamp, ...) are first and foremost integers.

Reply:
I think that to be 100% correct, `_sanitizeTimestamp()` should remain, and should call `_normalizeInt()`. Here, and everywhere else. Another question is that it's additional dev time, but it's still better. The reason for this is that a "timestamp" is not a synonym of "integer", and therefore their definitions may be different. Currently they are the same, but that may change.

Reply:
Agree. What about renaming to `_normalizeTimestamp()` instead?

Reply:
Sure.